### PR TITLE
Improved action loading and error message on name conflicts

### PR DESCRIPTION
**Pull request by KrauseFx**
- Refactored how actions are loaded to a central point
- Added warning when a lane name is already taken by an action

### CODE REVIEW

**Comment by KrauseFx** [comment]
Ready to be merged, however please don't squash, as these are 2 clearly separate commits 👍

**Comment by neonichu** [comment]
:-1:

File: fastlane/spec/actions_helper_spec.rb [hunk 1]
```diff
@@ -24,6 +24,12 @@
       end
     end
 
+    it "#action_class_ref" do
+      expect(Fastlane::Actions.action_class_ref("gym")).to eq(Fastlane::Actions::GymAction)
+      expect(Fastlane::Actions.action_class_ref(:cocoapods)).to eq(Fastlane::Actions::CocoapodsAction)
+      expect(Fastlane::Actions.action_class_ref('notExistentObv')).to eq(nil)
+    end
+
```
Comment (neonichu) [inline fastlane/spec/actions_helper_spec.rb]: tests, so useless 👎

**Comment by neonichu** [comment]
:+1:

**Pull request merged**